homeassistant/components/modern_forms/fan.py

```diff
@@ -142,35 +140,35 @@ async def async_turn_on(
             data[OPT_SPEED] = round(
                 percentage_to_ranged_value(self.SPEED_RANGE, percentage)
             )
-        await self.coordinator.modernforms.fan(**data)
+        await self.coordinator.modern_forms.fan(**data)
 
     @modernforms_exception_handler
     async def async_turn_off(self, **kwargs: Any) -> None:
         """Turn the fan off."""
-        await self.coordinator.modernforms.fan(on=FAN_POWER_OFF)
+        await self.coordinator.modern_forms.fan(on=FAN_POWER_OFF)
 
     @modernforms_exception_handler
     async def async_set_fan_sleep_timer(
         self,
         sleep_time: int,
     ) -> None:
         """Set a Modern Forms light sleep timer."""
-        await self.coordinator.modernforms.fan(sleep=sleep_time * 60)
+        await self.coordinator.modern_forms.fan(sleep=sleep_time * 60)
 
     @modernforms_exception_handler
     async def async_clear_fan_sleep_timer(
         self,
     ) -> None:
         """Clear a Modern Forms fan sleep timer."""
-        await self.coordinator.modernforms.fan(sleep=CLEAR_TIMER)
+        await self.coordinator.modern_forms.fan(sleep=CLEAR_TIMER)
 
 
 @callback
 def async_update_fan(
```

Comment:
We can inline this function into `async_setup_entry`. We only add entities during setup for this integration.

Reply:
moved inline